src/Chapter19/Listing19.07.RegisteringForUnhandledExceptions.cs

```diff
@@ -6,12 +6,12 @@
 
     public class Program
     {
-        public static Stopwatch _Clock = new Stopwatch();
+        private static Stopwatch _Clock { get;  } = new Stopwatch();
```

Comment:
We should fix the name of the property here. 
```suggestion
        private static Stopwatch Clock { get; } = new Stopwatch();
```